Classes/OGVDecoderWebM.m

```diff
@@ -267,6 +289,44 @@ -(BOOL)processBegin
                 }
             }
 #endif
+#ifdef OGVKIT_HAVE_OPUS_DECODER
+            if (audioCodec == NESTEGG_CODEC_OPUS) {
+                opusSampleRate = (opus_int32) audioParams.rate;
+                opusChannels = audioParams.channels;
+                int opusErr;
+                assert(opusDecoder == NULL);
+                opusDecoder = opus_decoder_create(opusSampleRate, opusChannels, &opusErr);
+                if (opusErr != OPUS_OK) {
+                    NSLog(@"Unable to initialize opus decoder with %@Hz and %@ channels. Error %d", opusSampleRate, opusChannels, opusErr);
```

Comment:
"%@Hz and %@ channels" should be "%dHz and %d channels" I think as these are ints